content/stable-rc.html

```diff
@@ -32,27 +32,10 @@
       -->
     </dd> 
     
-    <dt>Or native package</dt> 
-    <dd class="release-block-platforms"> 
-      <div> 
-        <img src="/images/os/win_other.png"> 
-        <a class="release-block-soft" href="http://mirrors.jenkins-ci.org/windows-stable-rc/latest">Windows</a> 
-      </div> 
-      <div> 
-        <img src="/images/os/ubuntu.png"> 
-        <a class="release-block-soft" href="http://pkg.jenkins-ci.org/debian-stable-rc/">Ubuntu/Debian</a> 
-      </div> 
-      <div> 
-        <img src="/images/os/redhat.png"> 
-        <a class=release-block-soft href="http://pkg.jenkins-ci.org/redhat-stable-rc/">Red Hat/Fedora/CentOS</a> 
-      </div>
-      <div> 
-        <img src="/images/os/os_macosx.png"> 
-        <a class=release-block-soft href="http://mirrors.jenkins-ci.org/osx-stable-rc/latest">Mac OS X</a> 
-      </div>
-      <div> 
-        <img src="/images/os/opensuse.png"> 
-        <a class=release-block-soft href="http://pkg.jenkins-ci.org/opensuse-stable-rc/">openSUSE</a> 
-      </div> 
-    </dd> 
-  </dl> 
+    <dt>native package</dt>
+    <dd class="release-block-platforms">
+        <div>
+            <p>native packages are not built anymore for LTS Release candidates.</p>
```

Comment:
Should just be left out. It's been so long (~two years) that nobody that used them should still have an expectation they still exist.